### PR TITLE
Adjustments for 21

### LINKED ISSUE #93: Migrate to @nextcloud/ packages
Please migrate to new @nextcloud/vue library and associated packages

### PR DESCRIPTION
**Pull request by blizzz**
- deprecations, style, DI
- npm deps, fixes #93 
- CI

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
+	"autoload-dev": {
+		"psr-4": {
+			"OCP\\": "vendor/christophwurst/nextcloud/OCP",
```
Comment (ChristophWurst) [inline composer.json:4]: why? the package provides that

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+			"OCA\\Talk\\": "lib/"
```
Comment (ChristophWurst) [inline composer.json:5]: this doesn't seem right?!

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+	"scripts": {
+		"cs:check": "php-cs-fixer fix --dry-run --diff",
+		"cs:fix": "php-cs-fixer fix",
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l"
+	},
     "require-dev": {
-        "php-parallel-lint/php-parallel-lint": "^1.2"
-    }
+		"nextcloud/coding-standard": "^0.5.0",
+		"christophwurst/nextcloud": "dev-master"
```
Comment (ChristophWurst) [inline composer.json:15]: as that would lock it to a certain commit I would say don't add this here but install only dynamically on CI

File: composer.json (hunk 1 above)
Comment (blizzz) [reply composer.json:4]: Copypasta

File: composer.json (hunk 2 above)
Comment (blizzz) [reply composer.json:5]: Oversaw that

File: composer.json (hunk 3 above)
Comment (blizzz) [reply composer.json:9]: why would that be locked? Only the JSON is shipped.

Comment (ChristophWurst) [reply composer.json:9]: then scratch what I said. I simply assume the lock file would be in git as well :zipper_mouth_face:

**Review by ChristophWurst — APPROVED** [review]

File: composer.json (hunk 3 above)
Comment (blizzz) [reply composer.json:9]: yeah let me include this

**Pull request merged**